spotbugs/src/main/java/edu/umd/cs/findbugs/detect/CbeckMustOverrideSuperAnnotation.java

```diff
@@ -36,29 +34,23 @@
 
 public class CbeckMustOverrideSuperAnnotation extends OpcodeStackDetector {
```

Comment:
Is this misspelled?

Reply:
I took that code (and that name) from a findbugs detector. You mean the "Cbeck" prefix? I don't know what that means. What would be a good name for this detector?

Reply:
"Cbeck" -> "Check" ?

Reply:
Oh, I see! Now I would call the detector "OverridingMethodsMustInvokeSuperDetector". The annotation suffix was misleading, too.